Make.inc

```diff
@@ -1417,7 +1417,7 @@ ifeq (,$(findstring aarch64,$(ARCH)))
 OSLIBS += -lgcc_s
 endif
 
-OSLIBS += -Wl,--export-dynamic -Wl,--undefined-version -Wl,--version-script=$(BUILDROOT)/src/julia.expmap \
+OSLIBS += -Wl,--export-dynamic -Wl,--version-script=$(BUILDROOT)/src/julia.expmap \
```

Comment:
I added `-Wl,--undefined-version` here as it was needed for FreeBSD AArch64. I'll build this PR on that system and see if your changes also fix the issue I hit.

Reply:
Yep seems to work